src/lib/Runner.ts

```diff
@@ -415,6 +415,7 @@ export class Runner {
     this.checkWaitingLandRequests();
   };
 
+  // TODO: CHECK FOR LOCK
```

Comment:
This one still todo?

Reply:
oops nope, that is only called by checkWaitingLandRequests so it's fine